webpack/assets/javascripts/dashboard/index.js

```diff
@@ -2,6 +2,7 @@ import $ from 'jquery';
 import { doesDocumentHasFocus } from '../react_app/common/document';
 import { notify } from '../foreman_toast_notifications';
 import { activateTooltips, foremanUrl } from '../foreman_tools';
+import { refreshPage } from '../foreman_navigation';
```

Comment:
I think you meant this:
```suggestion
import { reloadPage } from '../foreman_navigation';
```